catalog-rest-service/src/main/resources/ui/src/components/my-data/MyDataHeader.tsx

```diff
@@ -63,40 +88,62 @@ const MyDataHeader: FunctionComponent<Props> = ({
 
   useEffect(() => {
     setdataSummary({
```

Comment:
This object should be coming from a method,
Also while initialising state at top, we can set this state to `{}`, as this useEffect will be executed on mount and set these values before rendering

Reply:
Ohh, I missed this. will change it.